ethstore/src/json/key_file.rs

```diff
@@ -160,7 +160,7 @@ impl<'a> Visitor<'a> for KeyFileVisitor {
 
 		let address = match address {
 			Some(address) => address,
-			None => return Err(V::Error::missing_field("address")),
+			None => H160::zero(),
```

Comment:
this is unacceptable and may lead to errors

Reply:
ok, so I propose import with optional password to recover address when it is not in keystore